spec/features/posts_index_by_user_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe 'User Post Index', type: :feature do
```

Comment:
- [ ] you are still missing a few tests like `Tests that asserts you can see the user's profile picture.`, `Tests that asserts you can see how many comments a post has`, `Tests that asserts you can see how many likes a post has`, `Tests that asserts you can see the first comments on a post.`

Reply:
@VICTOR-CHUKWU User profile picture wasn't added to this section of my app so you want me to add it now before adding tests to it?